erts/doc/src/erl_dist_protocol.xml

```diff
@@ -1513,6 +1536,247 @@ DiB == gen_digest(ChA, ICA)?
+	  <seeguide marker="erl_dist_protocol#dflags">distribution flag</seeguide>.
+	</p>
+      </item>
+      <tag><marker id="UNLINK_ID_ACK"/><c>UNLINK_ID_ACK</c></tag>
+      <item>
+        <p><c>{36, Id, FromPid, ToPid}</c></p>
+        <p>An unlink acknowledgement signal. This signal is sent as an
+	acknowledgement of the reception of an 
+	<seeguide marker="#UNLINK_ID"><c>UNLINK_ID</c></seeguide>
+	signal. The <c>Id</c> element should be the same <c>Id</c>
+	as present in the <c>UNLINK_ID</c> signal. <c>FromPid</c>
+	identifies the sender of the <c>UNLINK_ID_ACK</c> signal and
+	<c>ToPid</c> identifies the sender of the <c>UNLINK_ID</c>
+	signal.</p>
+	<p>
+	  This signal is only passed when the 
+	  <seeguide marker="#new_link_protocol">new link protocol</seeguide>
+	  has been negotiated using the
+	  <seeguide marker="erl_dist_protocol#DFLAG_UNLINK_ID"><c>DFLAG_UNLINK_ID</c></seeguide>
+	  <seeguide marker="erl_dist_protocol#dflags">distribution flag</seeguide>.
+	</p>
+      </item>
     </taglist>
   </section>
+  <section>
+    <marker id="link_protocol"/>
+    <title>Link Protocol</title>
+
+    <section>
+      <marker id="new_link_protocol"/>
+      <title>New Link Protocol</title>
+
+      <p>
+	The new link protocol will be used when both nodes flag that
+	they understand it using the
+	<seeguide marker="#DFLAG_UNLINK_ID"><c>DFLAG_UNLINK_ID</c></seeguide>
+	<seeguide marker="#dflags">distribution flag</seeguide>. If
+	one of the nodes does not understand the new link protocol, the
+	<seeguide marker="#old_link_protocol">old link protocol</seeguide>
+	will be used as a fallback.
+      </p>
+      
+      <p>
+	The new link protocol introduces two new signals,
+	<seeguide marker="#UNLINK_ID"><c>UNLINK_ID</c></seeguide> and
+	<seeguide marker="#UNLINK_ID"><c>UNLINK_ID_ACK</c></seeguide>,
+	which replace the old
+	<seeguide marker="#UNLINK"><c>UNLINK</c></seeguide>
+	signal. The old <seeguide marker="#LINK"><c>LINK</c></seeguide>
+	signal is still sent in order to set up a link, but handled
+	differently upon reception.
+      </p>
+
+      <p>
+	In order to set up a link, a <c>LINK</c> signal is sent, from
+	the process initiating the operation, to the process that it
+	wants to link with. In order to remove a link, an
+	<c>UNLINK_ID</c> signal is sent, from the process initiating
+	the operation, to the linked process. The receiver of an
+	<c>UNLINK_ID</c> signal responds with an <c>UNLINK_ID_ACK</c>
+	signal. Upon reception of an <c>UNLINK_ID</c> signal, the
+	corresponding <c>UNLINK_ID_ACK</c> signal <em>must</em> be
+	sent before any other signals are sent to the sender of the
+	<c>UNLINK_ID</c> signal. This together with
+	<seeguide marker="erts:communication#passing-of-signals">the
+	signal ordering guarantees</seeguide> of Erlang makes it
+	possible for the sender of the <c>UNLINK_ID</c> signal to know
+	the order of other signals which is essential for the protocol.
```

Comment:
```suggestion
	the order of other signals which are essential for the protocol.
```

Reply:
Hm, I think it should be "is" here. It refers to "the order" not "signals".